.claude/commands/pgmq/update-docs.md

```diff
@@ -1,290 +0,0 @@
-# Sync Documentation and README with Source Code
```

Comment:
Is this causing a problem? @jumski recently added it and I've found it to be useful.